source/deployment/on-boarding.rst

```diff
@@ -67,3 +67,77 @@ Common Tasks
 **Upgrading Mattermost**
```

Comment:
This is now duplicated?

Reply:
Perhaps we just pull any unique info from here into below?

Reply:
Very good point. Thanks!